### PR TITLE
Stop eating fractions.

### PR DESCRIPTION
**Pull request by mtogstad**
This fixes address handling for house numbers and streets with fractions in them.

### CODE REVIEW

**Comment by adamolien** [comment]
@mtogstad just curious how much of a problem is. Do you have any numbers on how many properties we have with fractions? How many cases it was causing in issue in?

Also, can you think of any downsides to this?

File: lib/street_address/us.rb [hunk 1]
```diff
@@ -247,7 +245,7 @@ class << self
         (?: (?:#{unit_prefix_numbered_regexp} \W*)
             | (?<unit_prefix> \#)\W*
         )
-        (?<unit> [\w-]+)
+        (?<unit> [\w\/-]+)
```
Comment (adamolien) [inline lib/street_address/us.rb:248]: @mtogstad what does this change?

Comment (mtogstad) [reply lib/street_address/us.rb:248]: This allows a "/" in the "unit" section
e.g. 1213 w mifflin st unit b/101

**Comment by mtogstad** [comment]
@adamolien ~2000 at the moment, it happens regularly with duplexes and 2-on-a-lot houses. I think it is worth handling, especially considering the old behavior was to just eat them, potentially confusing one house with another house on the same lot.  There are also quite a few xx 1/2 streets out there, in which case you would placing a building on a parallel street by mistake.

**Comment by adamolien** [comment]
@mtogstad one final question... you mentioned duplexes, in that case wouldnt we want consider those a single property?

**Review by adamolien — APPROVED** [review]

**Comment by mtogstad** [comment]
@adamolien they could potentially be the same structure, but would still be different mailing addresses.

**Pull request merged**